dotcom-rendering/src/model/enhanceTreats.ts

```diff
@@ -1,7 +1,64 @@
 import type { FEFrontCard, TreatType } from '../types/front';
 
-export const enhanceTreats = (treats: FEFrontCard[]): TreatType[] =>
-	treats.map((treat) => ({
+/**
+ * PLATFORM_TREATS
+ *
+ * 👉 Edit this list below to change, remove or add new treats to front pages 👈
+ *
+ * This list controls treats that will appear in the bottom of the left column for a
+ * particular container. The actual container that a treat appears in is decided
+ * based on the values of displayName and editionId
+ *
+ * If both imageUrl and altText are provided, then an image will be shown. Otherwise,
+ * if no imageUrl or altText are given, then a basic text treat will be shown
+ * instead
+ */
+const PLATFORM_TREATS: TreatType[] = [
+	{
+		linkTo: '/info/2015/dec/08/daily-email-us?INTCMP=gdnwb_treat_election_today_us',
+		displayName: 'Spotlight',
+		editionId: 'US',
+		imageUrl:
+			'https://uploads.guim.co.uk/2020/10/22/newsletter-treat-img.png',
+		altText: 'The White House',
+		text: 'Guardian Today US: Get the headlines & more in a daily email',
+	},
+	{
+		linkTo: '/tv-and-radio/ng-interactive/2022/aug/01/whats-on-netflix-and-amazon-this-month-august',
+		displayName: 'Culture',
+		editionId: 'UK',
+		imageUrl:
+			'https://interactive.guim.co.uk/thrashers/culture-nugget/hashed/thrasher_img_55.1c0762e5.png',
+		altText: "What's on Netflix and Amazon this month",
+		text: "What's on Netflix & Amazon this month",
+	},
+];
+
+const getPlatformTreats = (
+	editionId: EditionId,
+	displayName: string,
+): TreatType[] => {
+	return PLATFORM_TREATS.filter((treat) => {
+		// We decide if a treat should be shown for a container based on
+		// if the editionId matches and if the displayName text is the same.
+		return (
+			treat.displayName === displayName && treat.editionId === editionId
```

Comment:
I'm not entirely sure this check will work as intended, I think with how its setup at the moment it would add the Culture treat to the first container of this [page](https://www.theguardian.com/uk/culture) which doesn't currently have a Culture treat in Frontend.

Could we use something like the PageId or the webUrl instead of editionId?

Reply:
> I think with how its setup at the moment it would add the Culture treat to the first container of this [page](https://www.theguardian.com/uk/culture) which doesn't currently have a Culture treat in Frontend

This is a good point. I didn't think about this.

But maybe this is a good thing? That container is very similar to the Culture one on the main front and having treats show in all places where 'Culture' shows is arguably correct.

Reply:
> But maybe this is a good thing? That container is very similar to the Culture one on the main front and having treats show in all places where 'Culture' shows is arguably correct.

Quite possibly! I suppose thats a question for editorial though!

Reply:
I'd definitely lean toward pageId/webUrl for the time being - otherwise the way these are controlled may end up being quite flakey

Reply:
So replace editionId = "UK" with pageId = "https://www.theguardian.com/uk"?

This would tighten the targeting

Reply:
Does that work across environments, I wondered if there was a relative `/uk` string as part of the data somewhere?

Reply:
After chatting with Harry, I'm proposing a more advanced solution: You can now choose to match on `containerName`, `editionId` OR `pageId`. This allows the culture treat to be shown for all Culture containers, if this is desired, or it can be targeted to one specific page